version.md

```diff
@@ -1 +0,0 @@
-Quest Guide <a href="/releasenotes.html">v2.5.1</a>
```

Comment:
@orcutt989 Just one change required here, I think - can you reinstate this file and bump the version to `2.5.2`? This file is pulled in by the markdown files during the `gitbook build` process. `releasenotes.md` should also include a new bullet point for 2.5.2 describing that JP language support has been dropped.